test/unit/.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "extends": "plugin:jasmine/recommended",
+  "plugins": [
+    "jasmine"
+  ],
+  "env": {
+    "node": true,
+    "jasmine": true,
+    "es6": true
+  },
+  "rules": {
+    "max-len": 0,
+    "complexity": 0,
+    "max-params": 0,
+    "max-statements": 0,
+    "jasmine/new-line-before-expect": 0,
+    "jasmine/new-line-between-declarations": 0,
+    "jasmine/no-suite-dupes": [1, "branch"],
+    "jasmine/no-spec-dupes": [1, "branch"],
+  
+    // Consider enabling these
```

Comment:
the downside of using `json` is we cannot add comment which makes the JSON file invalid. It seems mysterious how those tool be able to parse a invalid JSON file.

Reply:
that's a very interesting point!
👀